File: src/main/java/net/slipp/repository/qna/AnswerRepository.java
package net.slipp.repository.qna;

import net.slipp.domain.qna.Answer;
import net.slipp.domain.user.SocialUser;
import net.slipp.support.jpa.SlippCommonRepository;

import java.util.List;

public interface AnswerRepository extends SlippCommonRepository<Answer, Long> {
	List<Answer> findByWriter(SocialUser writer);
}

File: src/main/scala/net/slipp/repository/qna/AnswerRepository.scala
package net.slipp.repository.qna

import java.lang.Long
import java.util.List

import net.slipp.domain.qna.Answer
import net.slipp.domain.user.SocialUser
import org.springframework.data.jpa.repository.JpaRepository

trait AnswerRepository extends JpaRepository[Answer, Long] {
  def findByWriter(writer: SocialUser): List[Answer]
}